vep-mapping-pipeline/repeat_expansion_variants/pipeline.py

```diff
@@ -139,7 +163,7 @@ def determine_repeat_type(row):
         # For protein HGVS notation, assume that repeat is a trinucleotide one, since it affects entire amino acids
         repeat_type = 'trinucleotide_repeat_expansion'
     else:
-        # As a priority, use the repeat unit length determined directly from base sequence
+        # As a priority, use the repeat unit length determined directly from the HGVS-like base sequence
```

Comment:
Yes, I quite like this suggestion! In fact, maybe one day the repeat expansion pipeline can cease to exist as as separate entity, and this will be just handled by the ClinVar parser classes.

I've added a ticket to address this in future iterations: https://github.com/EBIvariation/eva-opentargets/issues/198